packages/styles/scss/components/lightbox-media-viewer/_lightbox-media-viewer.scss

```diff
@@ -23,6 +23,7 @@
       img {
         max-width: 100%;
         height: auto;
+        object-fit: cover;
```

Comment:
I'm going to go ahead and approve this, but I'm not sure this is needed since `Image` already has it. I'll approve this, but it might be worth taking a second look at this. @mkothur

Reply:
yeah, sorry I forgot to remove will update this.